spec/migrations/20220202122417_add_parent_id_to_miq_requests_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_migration
+
+describe AddParentIdToMiqRequests do
+  let(:miq_request) { migration_stub(:MiqRequest) }
+
+  migration_context :up do
+    it "adds parent_id to  Miq Requests" do
+      migrate
+      miq_request.create!(:type => 'MiqRequest', :parent_id => 40)
```

Comment:
You shouldn't use hard-coded IDs in schema specs, e.g. this ID is in region 0 and the region is randomized in spec tests.  It is better to actually create the other record and use that ID

Reply:
Hey look at that we have a spec for this https://github.com/ManageIQ/manageiq-schema/runs/5067896452?check_suite_focus=true#step:6:47